app/views/sponsors/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%div
+  %h3
+    Thank you to our Sponsors
+    .sponsors-temp
+      If you’re interested in sponsoring, check out our
+      = link_to "Sponsorship Prospectus", "#"
+      or email us at
+      = mail_to "sponsors@rubycentral.org", "sponsors@rubycentral.org"
```

Comment:
I wonder if we should be hard-coding these emails and maybe the content in general since this is still an open source repo. It sure would be cool if some other conference decided to use this part of cfp-app.

Reply:
I pulled this out entirely.